src/smif/sample_project/config/model_runs/energy_central.yml

```diff
@@ -11,7 +11,11 @@ narratives:
   technology:
   - high_tech_dsm
 strategies:
-- strategy: pre-specified-planning
+- name: pre-specified-planning
```

Comment:
I would suggest using a different tag here, such as 'type'. Name suggests a description of the strategy that is defined here. While pre-specified-planning or agent is more of a strategy-type.

Reply:
Good idea, changing to `type`